README.md

```diff
@@ -31,20 +25,122 @@ Download
 Download [the latest JAR][2] or grab via Maven:
 ```xml
 <dependency>
-  <groupId>com.squareup</groupId>
+  <groupId>com.squareup.tape2</groupId>
   <artifactId>tape</artifactId>
-  <version>1.2.3</version>
+  <version>2.0.0-SNAPSHOT</version>
 </dependency>
 ```
 or Gradle:
 ```groovy
-compile 'com.squareup:tape:1.2.3'
+compile 'com.squareup.tape2:tape:2.0.0-SNAPSHOT'
 ```
 
 Snapshots of the development version are available in [Sonatype's `snapshots` repository][snap].
 
 
 
+Usage
+--------
+
+Create a `QueueFile` instance.
+
+```java
+File folder = context.getDir("app", Context.MODE_PRIVATE);
```

Comment:
I was debating this. I'm hoping most developers recognize any folder can be used but still have something to copy paste for the more complex case (android)

I think the options are:

* java only example. `File folder = new File("app");`
* android only example (what's currently provided).
* Show both.

```java
// For Java apps.
File folder = new File("app");

// For Android apps.
File folder = context.getDir("app", Context.MODE_PRIVATE);
```

* Show neither and assume folder is defined somewhere.